tvbwidgets/core/pse/parameters.py

```diff
@@ -245,6 +249,16 @@ def _init_checkpoint(self):
     def monitor_execution(self):
         if self.update_progress is not None:
             self.update_progress()
+        else:
+            with self.progress_file_lock:
+                with open(PROGRESS_BAR_STATUS_FILE, "r+") as f:
+                    # set the cursor to the beginning of the file
+                    f.seek(0)
+                    status = int(f.read())
+                    status += 1
+                    f.seek(0)
+                    f.write(str(status))
```

Comment:
Should we maybe be extra safe and wrap the whole code inside `monitor_execution(self)` in a try/except, and go further if the progress can not be monitored, so that the jobs are not affected by this part ?